build(js): Change `app` alias to `sentry`

billyvg opened the pull request:
This changes the `app` alias we use for imports (that points to `static/app`) to `sentry`.  This makes it consistent with `getsentry` aliases.

Follow-up:
- [ ] Remove `app` alias from `getsentry`

priscilawebdev commented:
I like the idea. I am just wondering if prefixing it with `@` would be a good idea

BYK commented:
Bump?

billyvg commented:
I will get back to this around xmas break when activity dies down as it's hard to keep up

billyvg commented:
| find | replace |
| ---- | ------- |
| `\} from 'app\/` | `} from 'sentry\/` |
| `/* webpackChunkName: "(.*)" \*\/ 'app\/` | `/* webpackChunkName: "$1" */ 'sentry/` |

evanpurkhiser commented:
Gonna try and do it this xmas billy 👀

billyvg commented:
All I want for xmas...

billyvg commented:
I'll break these up into smaller PRs: https://github.com/getsentry/sentry/pull/30178

billyvg committed:
remove app alias

billyvg committed:
add git blame ignore revs

billyvg committed:
ref(ui): Change `.storybook` to use `sentry` alias

(These were added after initial refactoring PRs)
Part of refactoring from `app` import alias to `sentry`.

See https://github.com/getsentry/sentry/pull/30178

markstory reviewed (approved):
:clap:

billyvg committed:
vitalsCards
